modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,121 @@
+----
+.. Extract the `aarch64` VHD name and set it to `Blob_name`:
++
+[source,terminal]
+----
+Blob_name=rhcos-$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".release')-azure.aarch64.vhd
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container: 
++
+[source,terminal]
+----
+end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+sas=`az storage container generate-sas -n <container> --account-name <storage-account> --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container:
++
+[source, terminal]
+----
+az storage blob copy start --account-name $<storage-account> --sas-token "$sas" \
+ --source-uri "$RHCOS_vhd_origin_URL" \
+ --destination-blob "$Blob_name" --destination-container $<container>
+----
++
+You can check the status of the of copying process with the following command:
++
+[source,terminal]
+----
+"az storage blob show -c $<container> -n $Blob_name --account-name $<storage-account> | jq .properties.copy"
+----
++
+.Example output
+[source,terminal]
+----
+ "completionTime": null,
+ "destinationSnapshot": null,
+ "id": "1fd97630-03ca-489a-8c4e-cfe839c9627d",
+ "incrementalCopy": null,
+ "progress": "17179869696/17179869696",
+ "source": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-411.86.202207130959-0-azure.aarch64.vhd",
+ "status": "success", <1>
+ "statusDescription": null
+----
++
+<1> If the status parameter displays `success`, the copying process is complete. 
+ 
+. Create an image gallery using the following command:
++
+[source,terminal]
+----
+az sig create --resource-group <resource-group> --gallery-name <gallery> 
+----
+Use the image gallery to create an image definition:
++
+[source,terminal]
+----
+az sig image-definition create --resource-group <resource-group> --gallery-name <gallery> --gallery-image-definition rhcos-arm64 --publisher RedHat --offer arm --sku arm64 --os-type linux --architecture Arm64 --hyper-v-generation V2
+----
+. To get the URL of the VHD and set it to `RHCOS_vhd_url`, run the following command: 
++
+[source,terminal]
+----
+RHCOS_vhd_urlL=$(az storage blob url --account-name $<storage-account> -c $<container> -n "$Blob_name" -o tsv)
+----
+. Use the `RHCOS_vhd_url`, your storage account, resource group, and image gallery   to create an image version:
++
+[source,terminal]
+----
+az sig image-version create --resource-group $<resource-group> --gallery-name$<gallery> --gallery-image-definition rhcos-arm64 --gallery-image-version 1.0.0 --os-vhd-storage-account $<storage-account> --os-vhd-uri $RHCOS_vhd_url
+----
+. Your `arm64` imageID for the `resourceID` parameter is now generated: 
++
+.Example `resourceID` 
++
+[source,terminal]
+----
+/resourceGroups/$<resource-group>/providers/Microsoft.Compute/galleries/$<gallery>/images/$<image-name>/versions/$<image-version> 
```

Comment:
What does `image-name` refer to here?

Reply:
ah yes that would refer to the image-definition created in the previous step, i.e rhcos-arm64. would be good to reflect that